tests/baselines/reference/es3-jsx-react-native.types

```diff
@@ -4,8 +4,8 @@ const React: any = null;
 >null : null
 
 const elem = <div></div>;
->elem : error
-><div></div> : error
+>elem : any
+><div></div> : any

```

Comment:
Why did the printout of some of these ES3 error types change? Also, how was there an error type but no error baseline before? I feel pretty sure this isn’t a problem but I’m totally confused by it.

Reply:
Guessing: I believe configuration errors are supposed to effectively halt all future steps, so the checker is in an impossible-to-reach situation in this case